### ISSUE TITLE
Glitchy sorting animation on macOS

### ISSUE THREAD

**Issue by Luxbit** [yairEO/dragsort#5]
## Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### 💥 Demo Page

https://codepen.io/vsync/pen/jOqYOVJ?editors=1010

### Explanation
On macOS, the animation when sorting tags is glitchy, tested in newest version of Chrome, Safari and Firefox
It is working fine on Windows.

https://user-images.githubusercontent.com/2297843/116123235-0f45d380-a6c3-11eb-987f-a38b82484405.mov

- **What is the expected behavior?**
Smoothly animated sorting

- **What is happening instead?**
Tag jumps from origin to new spot on dragEnd

- **What error message are you getting?**
None

**Comment by fa-briz-io** [comment]
I came here because I was having the same problem. If anyone come with the same problem you can also use [SortableJS](https://github.com/SortableJS/Sortable)

```javascript
var el = document.getElementsByClassName('tagify');
var sortable = Sortable.create(el[0],{
  draggable: "."+tagify.settings.classNames.tag,
  onEnd: function (/**Event*/evt) {
    tagify.updateValueByDOMTags();
  }
});
```

**Comment by yairEO** [comment]
Yes, you can use any 3rd-party D&D library you wish in order to move tags around. Just remember to call `tagify.updateValueByDOMTags()` afterward every drop.

**Comment by one2gov** [comment]
I disabled animation in the line 287 dragover : this.throttle(this.dragover, 350) -> (dragover : this.throttle(this.dragover, 0))
And it feels much better

Demo
https://codepen.io/one2gov/pen/ExwLwae

**Comment by yairEO** [comment]
yeah I know, mac sucks :)

I use Windows and am enjoying life tremendously. 
I have 2 Mac and 2 Windows laptops and I simply cannot work with the the Macs , for years.. 
The OS and everything is too difficult to work with and there are many bugs

I remind you that this is a standalone package for sorting by dragging, and not intended for Tagify, although I am the author of both - you can use any other drag & sort package with Tagify. Just that they all suck (extremely bloated) :)
